packages/stream/solana/StreamClient.ts

```diff
@@ -344,29 +362,31 @@ export default class SolanaStreamClient extends BaseStreamClient {
    */
   public async createMultiple(
     data: ICreateMultipleStreamData,
-    { sender, isNative = false }: ICreateStreamSolanaExt
+    { sender, metadataPubKeys, isNative = false }: ICreateStreamSolanaExt
   ): Promise<IMultiTransactionResult> {
     const { recipients } = data;
 
     if (!sender.publicKey) {
       throw new Error("Sender's PublicKey is not available, check passed wallet adapter!");
     }
 
-    const metadatas: Keypair[] = [];
+    const metadatas: string[] = [];
     const metadataToRecipient: MetadataRecipientHashMap = {};
     const errors: ICreateMultiError[] = [];
     const signatures: string[] = [];
     const batch: BatchItem[] = [];
+    metadataPubKeys = metadataPubKeys || [];
 
-    for (const recipientData of recipients) {
-      const { tx, metadata } = await this.prepareStreamTransaction(recipientData, data, {
+    for (let i = 0; i < recipients.length; i++) {
+      const recipientData = recipients[i];
+      const { tx, metadataPubKey } = await this.prepareStreamTransaction(recipientData, data, {
         sender,
+        metadataPubKeys: metadataPubKeys[i] ? [metadataPubKeys[i]] : undefined,
```

Comment:
So we have 2 arrays: metadataPubKeys and recipients that have the same length.
Can we just assign both props in the single array?
Easier to itterate